src/Symfony/Component/DependencyInjection/Loader/Configurator/ContainerConfigurator.php

```diff
@@ -117,6 +117,16 @@ function iterator(array $values): IteratorArgument
  * Creates a lazy iterator by tag name.
  */
 function tagged(string $tag, string $indexAttribute = null, string $defaultIndexMethod = null): TaggedIteratorArgument
+{
+    @trigger_error('"tagged" tag is deprecated and will be removed in Symfony 5. Please use "tagged_iterator"', E_USER_DEPRECATED);
```

Comment:
Function "%s" is deprecated since Symfony 4.4 [...]

Reply:
Done.